Sources/IO/Core/HttpDataSetReader/test/testHttpDataSetReader.js

```diff
@@ -0,0 +1,222 @@
+import test from 'tape-catch';
+
+import vtkHttpDataSetReader from '..';
+import MockDataAccessHelper from '../../DataAccessHelper/MockDataAccessHelper';
```

Comment:
I recommend using the full import path `vtk.js/Sources/IO/Core/HttpDatasetReader` for consistency.

Reply:
I changed the paths in my most recent commit.